newproject2.php

```diff
@@ -1,11 +1,23 @@
 <?php
-$db = new PDO('mysql:host=db; dbname=Timelord_2021-02-08.sql', 'root', "password");
-$query = $db->prepare("query;");
+$db = new PDO('mysql:host=db; dbname=Timelord', 'root', "password");
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare('SELECT * FROM forgallifrey');
 $query->execute();
-$firstResult = $query->fetchAll();
-var_dump($firstResult);
+$forgallifreyArray = $query->fetchAll();
+//var_dump($forgallifreyArray);
+
+
+foreach ($forgallifreyArray as $item){
+   foreach($item as $key => $value){
```

Comment:
Chatting to Rayna, she told me it is good idea to label your variables to something related to your database. I had similar names for my arrays ($item as $key) then I changed them to $allbeers as $beerarray. It makes it easier to know what the variables are for.

Reply:
Cool. Good idea. Yep i've changes $item to $timelord now. Which is just so much more cool. Also if you don't already know, if you right click on a variable name, go to refactor and then rename and it will change the name of the variable wherever it turns up in your document so you don't have to change it multiple times :D